chain-tx-enclave-next/mls/src/ciphersuite.rs

```diff
@@ -77,12 +78,7 @@ impl Codec for ApplicationContext {
 }
 
 /// Additional methods for Kdf
-pub trait HkdfExt<D>
-where
-    D: Input + BlockInput + FixedOutput + Reset + Default + Clone,
-    D::BlockSize: ArrayLength<u8>,
-    D::OutputSize: ArrayLength<u8>,
-{
+pub trait HkdfExt {
```

Comment:
some comment about HkdftExt , can be replaced with fullword ?

Reply:
it has a comment "Additional methods for Kdf".
It expands the "Hkdf" trait with extra methods defined in: https://github.com/mlswg/mls-protocol/blob/master/draft-ietf-mls-protocol.md#key-schedule

Reply:
maybe it can reference it?

Reply:
done